### ISSUE TITLE
Mismatch Java version in sample apps and the library

### ISSUE THREAD

**Issue by sarimmehdi** [NordicSemiconductor/Android-BLE-Library#510]
I can't run any of your examples. Why is there a mismatch in your java versions? 

```
'compileDebugJavaWithJavac' task (current target is 1.8) and 'compileDebugKotlin' task (current target is 11) jvm target compatibility should be set to the same Java version.
```

**Comment by philips77** [comment]
Hi,
Sorry for that. We will fix it. Please, try [nRF Blinky](https://github.com/NordicSemiconductor/Android-nRF-Blinky) app which should work better.

**Comment by philips77** [comment]
Fixed with #516.
